.vscode/settings.json

```diff
@@ -26,5 +26,6 @@
     },
   ],
   "java.test.defaultConfig": "WPIlibUnitTests",
-  "java.compile.nullAnalysis.mode": "automatic"
+  "java.compile.nullAnalysis.mode": "automatic",
+  "wpilib.skipTests": true
```

Comment:
Is this set to true on purpose?

Reply:
yes, to avoid excessive delays during deploys from running unit tests, and instead my plan was to run these tests in ci